src/github.com/getlantern/errors/errors.go

```diff
@@ -276,15 +292,16 @@ func wrapSkipFrames(err error, skip int) Error {
 	}
 
 	// Create a new *structured
-	return buildError(err.Error(), err, cause)
+	return buildError("", "", err, cause)
```

Comment:
I noticed that by passing err.Error() here, we were effectively overriding the nice logic that @fffw built in `parseError`, so I fixed this while I was in here.